src/qt/guiutil.cpp

```diff
@@ -762,11 +762,11 @@ QString formatBytes(uint64_t bytes)
     if(bytes < 1024)
         return QString(QObject::tr("%1 B")).arg(bytes);
     if(bytes < 1024 * 1024)
-        return QString(QObject::tr("%1 KB")).arg(bytes / 1024);
+        return QString(QObject::tr("%1 KiB")).arg(bytes / 1024);
     if(bytes < 1024 * 1024 * 1024)
-        return QString(QObject::tr("%1 MB")).arg(bytes / 1024 / 1024);
+        return QString(QObject::tr("%1 MiB")).arg(bytes / 1024 / 1024);
 
-    return QString(QObject::tr("%1 GB")).arg(bytes / 1024 / 1024 / 1024);
+    return QString(QObject::tr("%1 GiB")).arg(bytes / 1024 / 1024 / 1024);
```

Comment:
I think this mebibyte, gibibyte naming scheme can be confusing to non-technical users